flink-core/src/test/java/org/apache/flink/api/common/typeutils/base/ListSerializerTest.java

```diff
@@ -28,7 +28,7 @@
 import java.util.Random;
 
 /** A test for the {@link LongSerializer}. */
```

Comment:
Probably not related to the main PR's goal however shouldn't it be 
```suggestion
/** A test for the {@link ListSerializer}. */
```
?